recipes/dicomifier/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "dicomifier" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lamyj/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4cf6e6711d0507c6a5b557e2cdf2199fe024d3e31b343c01c053ac1e5a3aef50
+
+build:
+  number: 0
+  skip: True  # [py<=35 or win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - boost-cpp
+    - odil
+    - pybind11
+    - python
+  run:
+    - boost-cpp
+    - nibabel
+    - numpy
+    - odil
+    - python
+    - python-dateutil
+
+test:
+  imports:
+    - dicomifier
+  commands:
+    - bruker2dicom --help
+    - dicom2nifti --help
+
+about:
+  home: https://github.com/lamyj/{{ name|lower }}
```

Comment:
Please don't use jinja templating in urls, here or for the `doc_url` and `dev_url`.

Reply:
Fixed in 7a362a10e. What about source URL?

Reply:
source is fine. I just prefer not to have them in the `home`/`dev_url`/`doc_url`, because when reviewing the recipe, those are links that I often cut and paste to look at the source to check that dependencies are correct, etc.